tsconfig.json

```diff
@@ -28,7 +28,7 @@
         "experimentalDecorators": true, // Enables experimental support for ES decorators
         "noFallthroughCasesInSwitch": true, // Report errors for fallthrough cases in switch statement
         "suppressImplicitAnyIndexErrors": true, // Index objects by number
-        "lib": ["dom", "es2019"]
+        "lib": ["dom", "es2021"]
```

Comment:
This is for `String.prototype.replaceAll` support in the types.

Reply:
Is `.replaceAll()` the only thing? As you mentioned in the other comment, it doesn't have ideal coverage yet, so I'd be in favor of keeping it out for now, which will simply make us use the universal `.replace(/foo/g, 'bar')` instead of `.replaceAll('foo', 'bar')`

Reply:
Missed this before merging :/.

That was indeed the only thing that required moving two years forward in time. Made a PR to revert this: https://github.com/PostHog/posthog/pull/9937